src/transmitter/index.ts

```diff
@@ -12,16 +12,40 @@ import {
   IDependencyGraphPayload,
 } from './types';
 import { getProxyAgent } from './proxy';
+import { queue } from 'async';
```

Comment:
nit: move library imports to the top of the file, please 🙏

Reply:
I refuse.


jk. Will do that now